dotnet-authserver/src/TeacherIdentity.AuthServer/IdentityLinkGenerator.cs

```diff
@@ -15,177 +15,180 @@ protected IdentityLinkGenerator(QueryStringSignatureHelper queryStringSignatureH
 
     protected QueryStringSignatureHelper QueryStringSignatureHelper { get; }
 
-    protected abstract string PageWithAuthenticationJourneyId(string pageName, bool authenticationJourneyRequired = true);
+    protected abstract string Page(string pageName, bool authenticationJourneyRequired = true);
```

Comment:
I've renamed this since it's also used for regular page links (without the authentication journey ID appended).